packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -851,6 +851,139 @@ void main() {
     expect(tester.getTopLeft(find.text('Item 0')).dx, 72);
     expect(tester.getTopLeft(find.text('Item 1')).dx, 72);
   });
+
+  test('PopupMenuButton takes only child or only icon', () {
+    // [PopupMenuButton] should assert that either child == null, icon == null,
+    // or that both are null because only one of the two can be displayed/used.
+    expect(() => PopupMenuButton<int>(
```

Comment:
Since it doesn't matter what the function returns, it's clearer to call it without the arrow function syntax. It is also only preferred if the return value fits on a single line.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods

```suggestion
    expect(() {
      PopupMenuButton<int>(
```